roles/virtualbmc/tasks/main.yml

```
@@ -18,7 +18,7 @@
   ansible.builtin.file:
     path: "{{ item.key }}"
     state: directory
-    mode: "{{ item.mode | default(omit) }}"
+    mode: "{{ item.mode | default("0755") }}"
```

```suggestion
    mode: "{{ item.mode | default('0755') }}"
```

Need to use single quotes inside double quotes

---

That should fix the pre-commit failure

---

and the virtualbmc failure as well :)

---

Thanks 🤦🏽‍♀️  I missed this